crates/neon-macros/src/export/function/mod.rs

```diff
@@ -145,6 +148,28 @@ pub(super) fn export(meta: meta::Meta, input: syn::ItemFn) -> proc_macro::TokenS
     .into()
 }
 
+fn to_camel_case(name: &str) -> String {
+    let mut should_upper = false;
+    let mut out = String::with_capacity(name.len());
```

Comment:
N.B.: This is a _reasonable_ heuristic to avoid allocations. In most cases it will over allocate by a few characters and in rare circumstances where `to_uppercase()` produces longer chars it will need to grow (TBH, I'm not even sure if this is possible with valid Rust identifiers).